.github/codeql/codeql-config.yml

```diff
@@ -0,0 +1,7 @@
+name: "Metabase CodeQL config"
+
+paths-ignore:
+  - "**/node_modules"
+  - "frontend/test/**"
+  - "frontend/**/*.unit.*"
+  - "e2e/**"
```

Comment:
https://docs.github.com/en/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors/customizing-code-scanning#specifying-directories-to-scan